Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -210,6 +213,7 @@ + (BranchActivityItemProvider *)getBranchActivityItemWithParams:(NSDictionary *)
 
 static BOOL bnc_useTestBranchKey = NO;
 static NSString *bnc_branchKey = nil;
+static BOOL bnc_disableFingerprintIDInCrashlyticsReports = NO;
```

Comment:
Can we call the flag `bnc_enableFingerprintIDInCrashlyticsReports` and have it default to `YES` instead?  I like keep methods positive so it's easier to reason about program logic and is consistent throughout the code.